src/App.js

```diff
@@ -0,0 +1,58 @@
+import CreateTodo from './components/CreateTodo.js'
+import DeleteTodo from './components/DeleteTodo.js'
+import EditTodo from './components/EditTodo.js'
+import FilterTodo from './components/FilterTodo.js'
+import CheckTodo from './components/CheckTodo.js'
+
+class App {
+  constructor () {
+    this.$newTodo = document.querySelector('#new-todo-title')
+    this.$todoList = document.querySelector('#todo-list')
+    this.$filters = document.querySelector('.filters')
+
+    this.createTodo = new CreateTodo(this.$newTodo, this.loadTodo)
+    this.deleteTodo = new DeleteTodo(this.$todoList, this.loadTodo)
+    this.editTodo = new EditTodo(this.$todoList, this.loadTodo)
+    this.filterTodo = new FilterTodo(this.$filters, this.updateTodo)
+    this.checkTodo = new CheckTodo(this.$todoList, this.loadTodo)
+
+    this.loadTodo()
+  }
+
+  loadTodo = () => {
+    this.todos = JSON.parse(localStorage.getItem('todos')) ?? []
+    this.updateTodo(this.todos)
+  }
+
+  updateTodo = (todos) => {
+    this.$todoList.innerHTML = ''
+    todos.map(
+      todo => this.$todoList.insertAdjacentHTML('beforeend', this.todoItemTemplate(todo))
+    )
+    this.updateTodoCount(todos)
+  }
+
+  todoItemTemplate = (todo) => {
+    return ` <li id=${todo.id} class=${todo.completed && 'completed'}>
+            <div class="view">
+              <input 
+                class="toggle" 
+                type="checkbox" 
+                id=${todo.id}
+                ${todo.checked && 'checked'}>
```

Comment:
```suggestion
            <input 
                class="toggle" 
                type="checkbox" 
                id=${todo.id}
                ${todo.checked && 'checked'}
              >```

가독성 측면에서 여는 괄호와 닫는 괄호가 동일한 위치에 있는 것이 좀 더 보기 편합니다.

Reply:
가독성을 신경쓰도록 노력하겠습니다💯